compiler/docgen.nim

```diff
@@ -840,9 +868,9 @@ proc genItem(d: PDoc, n, nameNode: PNode, k: TSymKind, docFlags: DocFlags) =
 
   let seeSrc = genSeeSrc(d, toFullPath(d.conf, n.info), n.info.line.int)
 
-  d.section[k].add(getConfigVar(d.conf, "doc.item") %
-    ["name", name, "uniqueName", uniqueName,
-     "header", result, "desc", comm, "itemID", $d.id,
```

Comment:
why was `"desc", comm` removed? (no need for a doc comment but just for review)

Reply:
doc comment `comm` changed type (was `string` before, became `PRstNode`) and cannot be added together with strings. It became a separate field `Item.descRst`.